README.md

```diff
@@ -1,29 +1,20 @@
-Python repository sample on how to build and layer your monorepo. 
-This is a cookiecutter reusable template and the steps bellow explain how to reuse this repository.
+# Cookiecutter Python repository demonstrating how to construct and layer your monorepo
 
-## Cookiecutter - create template for Python repository
+This is a lightweight, reusable Python Cookiecutter template, containing main folder structure you would need for any Python project.
 
-1. In your project, in order to create templates, rename all your files/folders with the keyword 'cookiecutter' followed by the item name
-    example: folder        -> {{cookiecutter.documents}}
-             python file   -> {{cookiecutter.main}}.py
-2. Add cookiecutter.json at the root of your project where you define the templates you created
-3. Push the folder structure to your repo (Github, Azure Devops)
+Project contains a basic Python folder structure, Architecture Decision Record & documentation structure, pull request template, as well as CI/CD pipeline with super-linter.
 
-## Cookiecutter - reuse template from Github (windows)
+## Quickstart
 
```

Comment:
Don't we wanna keep the steps on  'how to actually use the template' ? let me know what you think

Reply:
I compared it to some other cookiecutter templates, and think that the only thing that is actually needed is that cookiecutter http;//github/...repo and then git init -b main. I can add that last line still...